02/Task02.js

```diff
@@ -1,15 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Welcome from './../src/components/Welcome';
 import withPopup from './../src/hoc/withPopup';
 
-const ComponentWithPopup = withPopup(Welcome);
+const ComponentWithPopup = withPopup('darkgreen', 0.7)(Welcome);
 
-const Task02 = () => (
-    <section>
-        <h2>Task 02</h2>
-        <ComponentWithPopup />
-    </section>
-)
+const Task02 = () => {
+    const [isPopUpActive, setIsPopUpActive] = useState(true);
+    return (
+        <section>
+            <h2>Task 02</h2>
+            {isPopUpActive ? (
+                <ComponentWithPopup
+                    setIsPopUpActive={setIsPopUpActive}
+                    isPopUpActive={isPopUpActive}
+                />
+            ) : (
+                <Welcome />
+            )}
```

Comment:
Próbuj zrobić to tak, aby nie trzeba było robić warunku tutaj.
Cała logika ma być obsłużona przez `withPopup()`